sparse/_compressed/indexing.py

```diff
@@ -87,6 +87,8 @@ def getitem(x, key):
         elif isinstance(ind, np.ndarray) and ind.ndim > 1:
             raise IndexError("Only one-dimensional iterable indices supported.")
 
+        reordered_key[i] = reordered_key[i].astype(x.indices.dtype, copy=True)
```

Comment:
Why `copy=True` if we won't be mutating this?

Reply:
brainfart 😅 the default is `True` but I wanted to set it to `False` for when it was not needed.